.github/actions/upload-test-artifacts/action.yml

```diff
@@ -50,12 +50,8 @@ runs:
         rm -f logs-*.zip
         # this workflow is also run in bazel build test, but we dont generate usage reports for it
         # so check to see if the file exists first
-        if [ -f 'usage_log.txt' ]; then
-            zip "logs-${FILE_SUFFIX}.zip" 'usage_log.txt'
-        fi
-        if ls test/**/*.log 1> /dev/null 2>&1; then
-            zip -r "logs-${FILE_SUFFIX}.zip" test -i '*.log'
-        fi
+        zip "logs-${FILE_SUFFIX}.zip" 'usage_log.txt' || true
```

Comment:
I'm not sure if `|| true` is such a good idea, as it can mask other errors such as typo in the name of the command, some folder prefixes etc

Reply:
I had a bit of trouble with the if ls, I think it doesn't like the -, but switching to find instead now works